### PR TITLE
Connie branch

### PR DESCRIPTION
**Pull request by NicaVulcan**

### CODE REVIEW

File: character.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
```
Comment (ErikPlachta) [inline character.html:1]: That is great. Thank you for building this out

File: controllers/home-route.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const router = require("express").Router();
+
+router.get('/', (req, res) => {
+    res.render('homepage')
+
+})
+
```
Comment (ErikPlachta) [inline controllers/home-route.js:7]: Excited to start using this!

File: public/javascript/character.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+// user clicking option
```
Comment (ErikPlachta) [inline public/javascript/character.js:1]: Excited to start building this out!

File: views/character.handlebars [hunk 4]
```diff
@@ -0,0 +1,45 @@
+        <div class="character-creator">
```
Comment (ErikPlachta) [inline views/character.handlebars:1]: Excited to start using this :D

File: views/character.handlebars [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+                <option value="Elf">Elf</option>
+                <option value="Dawrf">Dwarf</option>
+                <option value="Human">Human</option>
+                <option value="Orc">Orc</option>
+            </select>
+
+                <!-- gender -->
+            <label for="gender-list">Gender: </label>
+            <select id="gender-list" v-model="gender">
+                <option value="Male">Female</option>
```
Comment (mmlawton15) [inline views/character.handlebars:16]: shouldnt "Female" be in the quotes here?

**Pull request merged**